src/main/java/com/icthh/xm/tmf/ms/activation/repository/SagaLogRepository.java

```diff
@@ -48,7 +48,7 @@ List<SagaLog> getLogsBySagaTransactionIdAndTypeKey(
     @Query("SELECT s FROM SagaLog s " +
         "WHERE (s.logType = 'REJECTED_BY_CONDITION' OR s.logType = 'EVENT_END') " +
         "AND s.sagaTransaction.id = :sagaTransactionId AND s.eventTypeKey = :eventTypeKey " +
-        "AND s.iteration = :iteration")
+        "AND (s.iteration = :iteration OR (s.iteration is null AND :iteration = null))")
```

Comment:
Just interesting
NVL(s.iteration, :iteration) = :iteration
will do same thing? Or for PG/Oracle we have to support NVL/COALESCE